docs/pages/cldvideoplayer/configuration.mdx

```diff
@@ -19,30 +19,76 @@ import OgImage from '../../components/OgImage';
 
 | Prop Name          | Type           | Default         | Description                              | Example                      |
 |--------------------|----------------|-----------------|------------------------------------------|------------------------------|
-| autoPlay           | string         | `"never"`       | When, if, should the video automatically play. See `autoplayMode` in [Video Player docs](https://cloudinary.com/documentation/video_player_api_reference#constructor_parameters) | `"on-scroll"`          |
```

Comment:
i think this fell out somewhere, i dont see it re-added anywhere

Reply:
im gonna add it back in when fixing the merge conflict

Reply:
nm looks like you fixed the mege conflict, if you want to make sure this is in there

Reply:
Ohh okay noticed it too, sorry about that. it will be included in my next commit